src/ccache.h

```diff
@@ -243,6 +245,7 @@ void wipe_all(struct conf *conf);
 int execute(char **argv, int fd_out, int fd_err, pid_t *pid);
 char *find_executable(const char *name, const char *exclude_name);
 void print_command(FILE *fp, char **argv);
+char *string_command(char **argv);
```

Comment:
I suggest renaming to `format_command` analogous to the other similar functions.

Reply:
Done.